File: GrocerioApi/Database/Entities/Store.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using GrocerioModels.Enums.Store;

namespace GrocerioApi.Database.Entities
{
    public class Store
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public Membership Membership { get; set; }
        public string ImageLink { get; set; }
        public int UniqueStoreNumber { get; set; }

        public List<StoreProducts> StoreProducts { get; set; }
    }
}

File: grocerio_mobile/lib/models/store.dart
class Store {
  int id;
  String name;
  String description;

  Store({
    this.id = 0,
    this.name = '',
    this.description = '',
  });

  factory Store.fromJson(Map<String, dynamic> json) {
    return Store(
      id: json['id'],
      name: json['name'],
      description: json['description'],
    );
  }
}
